### PR TITLE
draft

### PR DESCRIPTION
**Pull request by voloshynamaryna11**

### CODE REVIEW

File: src/main/resources/hibernate.cfg.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        <property name="hibernate.connection.url">jdbc:mysql://localhost:3306/cinema?serverTimezone=UTC</property>
+        <property name="hibernate.connection.username">root</property>
+        <property name="hibernate.connection.password">puv34g7hhe4jkopp2!1e$</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="cinema.model.Movie"/>
+    </session-factory>
+
+</hibernate-configuration>
```
Comment (KostiantynVorobiov) [inline src/main/resources/hibernate.cfg.xml]: empty line

**Review by KostiantynVorobiov — APPROVED** [review]
OK

File: pom.xml [hunk 2]
```diff
@@ -61,6 +71,14 @@
                     <webXml>src/main/webapp/web.xml</webXml>
                 </configuration>
             </plugin>
```
Comment (KseniiaMakarova) [inline pom.xml]: do you need maven war plugin?

File: pom.xml [hunk 3]
```diff
@@ -61,6 +71,14 @@
...
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
```
Comment (KseniiaMakarova) [inline pom.xml]: does it work without this plugin? you already have it in `<pluginManagement>` section

File: src/main/java/cinema/dao/impl/MovieDaoImpl.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            session.save(movie);
+            transaction.commit();
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't insert movie Entity", e);
+        } finally {
+            session.close();
```
Comment (KseniiaMakarova) [inline src/main/java/cinema/dao/impl/MovieDaoImpl.java]: let's add `if (session != null)` check

File: src/main/java/cinema/service/impl/MovieServiceImpl.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import cinema.lib.Inject;
+import cinema.lib.Service;
+import cinema.model.Movie;
+import cinema.service.MovieService;
+import java.util.List;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+    @Inject
+    MovieDao movieDao;
```
Comment (KseniiaMakarova) [inline src/main/java/cinema/service/impl/MovieServiceImpl.java]: access modifier

File: src/main/resources/hibernate.cfg.xml (hunk 1 above)
Comment (KseniiaMakarova) [inline src/main/resources/hibernate.cfg.xml]: add an empty line

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/cinema/dao/impl/MovieDaoImpl.java (hunk 4 above)
Comment (voloshynamaryna11) [reply src/main/java/cinema/dao/impl/MovieDaoImpl.java]: done

File: src/main/java/cinema/service/impl/MovieServiceImpl.java (hunk 5 above)
Comment (voloshynamaryna11) [reply src/main/java/cinema/service/impl/MovieServiceImpl.java]: done

File: src/main/resources/hibernate.cfg.xml (hunk 1 above)
Comment (voloshynamaryna11) [reply src/main/resources/hibernate.cfg.xml]: done

File: pom.xml (hunk 3 above)
Comment (voloshynamaryna11) [reply pom.xml]: done

File: pom.xml (hunk 2 above)
Comment (voloshynamaryna11) [reply pom.xml]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**